src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,148 @@
+* Libraries shared with the Dart community are distributed as library packages,
+  published at [pub.dartlang.org](https://pub.dartlang.org/), and documented at
+  [dartdocs.org](https://www.dartdocs.org/). The [pub](/tools/pub/) tool
+  allows you to create, publish, and manage library packages.
+* Libraries that are not shared with the community are simply placed in the
+  `/lib` directory of your [application's directory
+  structure](https://www.dartlang.org/tools/pub/package-layout#public-directories).
+
+This document talks about the first two kinds of libraries,
+and tells you where to learn more about some of the most widely used Dart libraries.
+
+<aside class="alert alert-info" markdown="1">
+If you don't find the functionality you need on [api.dartlang.org]({{site.dart_api}}),
+check [pub.dartlang.org](https://pub.dartlang.org/).
+</aside>
+
+## Dart SDK libraries
+
+Dart's core libraries contain classes most commonly used by developers.
+Classes that are still useful, but not as universally used,
+are placed in packages outside of the SDK.
+Separating out the less commonly used classes ensures that your app
+isn't bloated by features that you don't need.
+
+The following table includes the most commonly used libraries in the Dart SDK:
+
+| **SDK Library** | **Description** | **Example classes and members** |
+| [dart:async](https://api.dartlang.org/stable/dart-async/dart-async-library.html) | Support for asynchronous programming. | Future, Stream |
+| [dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html) | Classes and utilities that supplement the collection support in dart:core. | Queue, HashMap, HashSet |
+| [dart:convert](https://api.dartlang.org/stable/dart-convert/dart-convert-library.html) | Encoders and decoders for converting between different data representations, including JSON and UTF-8.| Converter, Base64Decoder, LineSplitter, UTF8Decoder |
+{:.table .table-striped .nowrap}
+
+Looking for web, server, or Flutter libraries?
+See [Specialized libraries](#specialized-libraries).
+
+## Corresponding Dart packages
+
+These "expansion pack" libraries have the same name as their counterpart in the SDK.
+Each of these libraries builds upon an SDK library, adding
+additional functionality and filling in missing features:
+
+| **Package** | **Description** | **Example classes and members** |
+| [async](https://www.dartdocs.org/documentation/async/latest/) | Utility classes to work with asynchronous computations. For more information, see [Unboxing Packages: async part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html), [part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html), and [part 3.](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html) | AsyncMemoizer, CancelableOperation, FutureGroup, LazyStream, Result, StreamCompleter, StreamGroup, StreamSplitter, !isCompleted  |
```

Comment:
Remove `!isCompleted` here. That was just Natalie's little joke in the blogpost.